docs/src/main/asciidoc/_migration-guide-for-4.0.adoc

```diff
@@ -1429,7 +1431,8 @@ IMPORTANT: The binder type is combined from `servicebus-queue` and `servicebus-t
 |*spring.cloud.stream.servicebus.topic*.bindings.<binding-name>.consumer.*checkpoint-mode* |*spring.cloud.stream.servicebus*.bindings.<binding-name>.consumer.*auto-complete*
 |===
 
-NOTE: The concurrency property will replace maxConcurrentSessions when sessionsEnabled is `true` and maxConcurrentCalls when sessionsEnabled is `false`. Enabling auto-complete is equal to `RECORD` checkpoint mode, and oppositely the `MANUAL` mode.
+NOTE: The concurrency property will be replaced by the maxConcurrentSessions when sessionsEnabled is `true` and the maxConcurrentCalls when sessionsEnabled is `false`.
+NOTE: Enabling auto-complete is equal to `RECORD` checkpoint mode, and oppositely the `MANUAL` mode.
```

Comment:
This format is not correct, 
<img width="1834" alt="image" src="https://user-images.githubusercontent.com/31124698/159826915-bcb21c7c-dc19-4771-bde2-2efc3d7f6015.png">
we expect the rendering effects before this change